finality-aleph/src/sync/mod.rs

```diff
@@ -28,7 +28,7 @@ pub use substrate::{
     SubstrateChainStatus, SubstrateChainStatusNotifier, SubstrateFinalizationInfo, VerifierCache,
 };
 
-const LOG_TARGET: &str = "aleph-block-sync";
+use crate::metrics::LOG_TARGET;
```

Comment:
Uhh, no, this shouldn't be using the metrics log target.

Reply:
:facepalm: must've removed that accidentally and then auto import chosen the only possible and incorrect version